### PR TITLE
Publish /shutdown topic if go-to-kitchen demo fails

### PR DESCRIPTION
**Pull request by 708yamaguchi**
From https://github.com/knorth55/jsk_robot/pull/169
(I don't cherry-pick the first two commits, because these commits are already merged.)

With this PR, fetch shuts down itself when `go-to-kitchen` demo fails by timeout.

The purpose of this PR is to prevent fetch battery degradation.

### CODE REVIEW

**Comment by knorth55** [comment]
this PR depends on https://github.com/PR2/app_manager/pull/49.
Waiting for upstream.

File: jsk_fetch_robot/jsk_fetch.rosinstall.melodic [hunk 1]
```diff
@@ -1,10 +1,12 @@
 # This is rosinstall file for melodic PC inside fetch.
 # $ ln -s $(rospack find jsk_fetch_startup)/../jsk_fetch.rosinstall.$ROS_DISTRO $HOME/ros/$ROS_DISTRO/src/.rosinstall
 
+# Use https://github.com/PR2/app_manager.git kinetic-devel branch
+# after https://github.com/PR2/app_manager/pull/49 is merged
```
Comment (knorth55) [inline jsk_fetch_robot/jsk_fetch.rosinstall.melodic:5]: it is already merged

**Review by knorth55 — CHANGES_REQUESTED** [review]

**Review by knorth55 — CHANGES_REQUESTED** [review]

File: jsk_fetch_robot/jsk_fetch.rosinstall.melodic [hunk 2]
```diff
@@ -58,10 +60,12 @@
...
     uri: https://github.com/knorth55/jsk_robot.git
     version: fetch15
+# Use https://github.com/knorth55/app_manager_utils master branch
+# after https://github.com/knorth55/app_manager_utils/pull/40 is merged
 - git:
     local-name: knorth55/app_manager_utils
-    uri: https://github.com/knorth55/app_manager_utils
-    version: master
+    uri: https://github.com/708yamaguchi/app_manager_utils
+    version: fetch15
```
[the remark this answers is not in the document]
Comment (708yamaguchi) [reply jsk_fetch_robot/jsk_fetch.rosinstall.melodic]: @knorth55 

Thank you for your review.

I updated `app_manager_utils` branch by cherry-pick (https://github.com/knorth55/jsk_robot/commit/c38f325eae6a2323ff6483be55bc9b54720f1e36)

I updated `app_manager` branch by cherry-pick (https://github.com/knorth55/jsk_robot/commit/a4957cbfb0710784593b570c155f680306c73d8a) and additional commit (https://github.com/jsk-ros-pkg/jsk_robot/pull/1452/commits/ea1a9b7dd5b081265c7958851b951794d26ded1c)

**Comment by 708yamaguchi** [comment]
I rebased `origin/master` to resolve conflicts.

**Pull request merged**